docs/fastdds/ros2/discovery_server/ros2_discovery_server.rst

```diff
@@ -318,43 +320,38 @@ As most of ROS 2 CLI Introspection is executed by adding a node into the network
 and some create their own nodes), using Discovery Server v2 we will find that most of these functionalities are
 limited and do not have all the information.
 
-The Discovery Server v2 functionality allows every node running as a **Server** (a kind of *Participant type*)
-to know and share all the participants and topics information with every other server matched.
-In this sense, a server can be configured alongside ROS 2 introspection, since then the introspection tool will
-be able to discover every entity in the network that is using the Discovery Server protocol.
+The Discovery Server v2 functionality allows every node running as a |SUPER_CLIENT|, a kind of *Participant type* that
```

Comment:
Is it not redundant to say `kind of Participant type`? If it is referring to a specific structure maybe it is better to link to the alias.